src/App.js

```diff
@@ -1,7 +1,70 @@
 import Background from "./assets/Background.svg";
 import Logo from "./assets/spacecenter1.svg";
+import { useNavigate } from "react-router-dom";
 
 function App() {
+  const LOGIN_API_URL = "http://localhost:3000/auth/login";
```

Comment:
Instead, I think it makes more sense to have an `API_URL` environment variable that you can set to `http://localhost:3000`, then in each call, you can use only the path, or set these consts.